Makefile

```diff
@@ -4,7 +4,6 @@ export
 install_deps:
 	pip install -r dev_requirements.txt
 	pip install -r requirements.txt
-	pip install -e .
```

Comment:
For now I remove this since these are the same requirements as in `requirements.txt`. I don't know if we want to keep `requirements.txt` if `setup.py` contains the same libraries.

The only value I can see is for some IDEs that can detect missing libraries from requirements.txt file (like PyCharm). 

But this issue might be tackled when using Poetry.

Reply:
I will suggest to still leave `pip install -e. ` as calling make install_deps can be called exactly as it is in the CI

Reply:
Done